Sources/Register.php

```diff
@@ -166,6 +166,16 @@ function Register($reg_errors = array())
 		$user_info['permissions'] = array_merge($user_info['permissions'], array('profile_account_own', 'profile_extra_own', 'profile_other_own', 'profile_password_own', 'profile_website_own', 'profile_blurb'));
 		$reg_fields = explode(',', $modSettings['registration_fields']);
 
+		// Website is a little different
+		if (in_array('website', $reg_fields))
+		{
+			unset($reg_fields['website']);
+			if (isset($_POST['website_title']))
+				$cur_profile['website_title'] = $smcFunc['htmlspecialchars']($_POST['website_title']);
+			if (isset($_POST['website_url']))
```

Comment:
This looks pretty good. However, we need the user to supply both the title and the url of their website, so it would be better to do a single check for both, like this:

```
if (!empty($_POST['website_title']) && !empty($_POST['website_url'])
{
	$cur_profile['website_title'] = $smcFunc['htmlspecialchars']($_POST['website_title']);
	$cur_profile['website_url'] = $smcFunc['htmlspecialchars']($_POST['website_url']);
}
```

Reply:
Please had in mind what the scope of this pr is.
The scope is that when you reg doing something wrong exp. you write the wrong auth code,
than you return to the reg form and this pr fix the issue that in this case the title and url are not empty -> we want also partly save.

Reply:
Yes, you are right. That makes sense then.